plugin/dnstap/dnstapio/io.go

```diff
@@ -13,34 +13,40 @@ import (
 const (
 	tcpTimeout   = 4 * time.Second
 	flushTimeout = 1 * time.Second
-	queueSize    = 1000
```

Comment:
We should remove this

Reply:
Without queue we will have "Dnstap payload dropped" all time.